lib/docpub.js

```diff
@@ -49,8 +51,8 @@ module.exports = class DocpubPipeline {
 
         return category.read()
             .then(() => {
-                const categoryUploader = new CategoryUploader(category);
-                const zendeskDeleter = new ZendeskDeleter(category);
+                const categoryUploader = new CategoryUploader(category, this._config);
```

Comment:
Actually, I wonder if the config really needs to be passed to any of the uploaders. All of the options that might need to be changed are part of the `apiUtils.getClient()` method. 

So you could create the zendesk client here (passing it the config) and then pass the client to the uploaders. This would also mean `zendeskClient` would become a required property for the uploaders.